extension/src/background_scripts/background.js

```diff
@@ -147,6 +156,9 @@ chrome.runtime.onMessage.addListener(
         case "MuteCurrentTab":
             handleMuteCurrentTab();
             sendResponse({ status: true });
+        case "FetchVideoFeedback":
```

Comment:
Move this base on the the alphabetical order please

Reply:
Done

Reply:
Done